rllib/core/learner/tf/tf_learner.py

```diff
@@ -95,6 +95,18 @@ def apply_gradients(self, gradients: ParamDictType) -> None:
             optim.apply_gradients(zip(gradient_list, variable_list))
 
     @override(Learner)
+    def postprocess_gradients(
+        self, gradients_dict: Mapping[str, Any]
+    ) -> Mapping[str, Any]:
+        grad_clip = self._optimizer_config.get("grad_clip", None)
```

Comment:
This doesn't seem to exist for as an option for all algorithms just yet. Otherwise it would probably be apart of algorithm_config options.

Going to leave as is since I moved the grad_clip configuration into impala.py from algorithm_config.py

Reply:
added a todo